docs/app/docs/devbox_examples/stacks/laravel.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Laravel
+description: Create a Laravel dev environment with PHP, MariaDB, and Redis
+---
+Laravel is a powerful web application framework built with PHP. It's a great choice for building web applications and APIs.
+
+This example shows how to build a simple Laravel application backed by MariaDB and Redis. It uses Devbox Plugins for all 3 Nix packages to simplify configuration
+
+[Example Repo](https://github.com/jetpack-io/devbox/tree/main/examples/stacks/laravel)
+
+[![Open In Devbox.sh](https://jetpack.io/img/devbox/open-in-devbox.svg)](https://devbox.sh/github.com/jetpack-io/devbox/?folder=examples/stacks/laravel)
+
+## How to Run
+
+1. Install [Devbox](https://www.jetpack.io/devbox/docs/installing_devbox/)
+
+1. Create a new Laravel App by running `devbox run create-project`. This will create a new Laravel project in your current directory.
```

Comment:
```suggestion
1. Create a new Laravel App by running `devbox create --template laravel`. This will create a new Laravel project in your current directory.
```